app/test_main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from typing import Any
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.has_internet_connection")
+def test_has_internet_connection_called(
+        mocked_has_internet_connection: Any
+) -> None:
```

Comment:
The test name `test_has_internet_connection_called` is not very descriptive. According to the checklist, use descriptive and correct test names. Consider renaming it to something like `test_can_access_google_page_calls_has_internet_connection`.